ClothesApp/WebAPI/Controllers/OrderController.cs

```diff
@@ -3,21 +3,24 @@
 using Application.Dtos.OrderTransactions;
 using Application.Interfaces.Services;
 using Domain.Enums;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
+using WebAPI.Authentication;
 
 namespace WebAPI.Controllers;
 
 [ApiController]
-[Route("api/orders")] 
+[Route("api/orders")]
 public class OrderController : ControllerBase
 {
-    private readonly IOrderService _orderService;   
-    
+    private readonly IOrderService _orderService;
+
     public OrderController(IOrderService orderService)
     {
         _orderService = orderService;
     }
-    
+
+    [Authorize]
```

Comment:
This policy is not precise, because customer can only have access to his orders, while admin has access to all